### DISCUSSION TITLE
Is it possible to know the AP the pico-w is connected to?

### DISCUSSION THREAD

**Discussion by joba-1** [earlephilhower/arduino-pico#1279] [Q&A] [answered]
is something like ESP’s BSSID() available on the Picos?

**Comment by earlephilhower** [comment, marked as answer]
Yes, and it's surprisingly called `WiFi.BSSID()` 😆 https://github.com/earlephilhower/arduino-pico/blob/54f9d3c4149d863ba24a2697b501098f95c261a1/libraries/WiFi/src/WiFiClass.h#L276

**Reply by joba-1** [reply]
sorry to have wasted your time on this! I should have found it myself when I looked in the wifi class header today - must have been blind and only found the scan version :(
